src/lib/Behat/Component/IbexaDropdown.php

```diff
@@ -43,4 +43,16 @@ public function selectOption(string $value)
         usleep(2000000);
         $listElement->find($this->getLocator('ibexaDropdownLabel'))->click();
     }
+
+    public function selectOptionByValueFragment(string $valueFragment)
```

Comment:
Not sure about this name, I'm open to suggestions 😄

Reply:
This name states clearly what it does imo.